src/components/ButtonNav/styled.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+
+export const Wrapper = styled.div`
```

Comment:
This should probably be `nav` element.
```suggestion
export const Wrapper = styled.nav`
```